libr/core/cmd_debug.c

```diff
@@ -3243,8 +3243,19 @@ static void r_core_cmd_bp(RCore *core, const char *input) {
 	case '*': r_bp_list (core->dbg->bp, 1); break;
 	case '\0': r_bp_list (core->dbg->bp, 0); break;
 	case '-': // "db-"
-		if (input[2] == '*') r_bp_del_all (core->dbg->bp);
-		else r_bp_del (core->dbg->bp, r_num_math (core->num, input + 2));
+		if (input[2] == '*') {
+			r_bp_del_all (core->dbg->bp);
+		} else {
+			#define DB_ARG(x) r_str_word_get0(str, x)
+			for (p = input + 2; *p == ' '; p++);
+			char *str = strdup (p);
+			int sl = r_str_word_set0 (str);
+			for (int i = 0; i < sl; i++) {
+				r_bp_del (core->dbg->bp, r_num_math (core->num, DB_ARG(i)));
```

Comment:
maybe better split this line in 2, const ut64 addr = r_num_math... and then rbpdel(addr)

Reply:
Thanks for review, I will fix these things, but, honestly speaking, most of these lines already were in the code in such condition, so it's not my fault.